openpype/lib/applications.py

```diff
@@ -1503,9 +1504,15 @@ def _prepare_last_workfile(data, workdir):
     project_name = data["project_name"]
     task_name = data["task_name"]
     task_type = data["task_type"]
-    start_last_workfile = should_start_last_workfile(
-        project_name, app.host_name, task_name, task_type
-    )
+
+    start_last_workfile = data.get("start_last_workfile", True)
+    if start_last_workfile:
+        start_last_workfile = should_start_last_workfile(
+            project_name, app.host_name, task_name, task_type
+        )
+    else:
+        log.info("Explicitly forbidden to open last workfile")
```

Comment:
Forbidden sounds like quite a harsh phrasing of what is happening. Isn't this a local setting to the user? :) Should we rephrase that to: "Explicitly disabled to open last workfile"? Just wondering.

Reply:
Agreed. I'd  go a bit more verbose and human. `Opening last workfile was disabled`

Reply:
Changed